src/index.ts

```diff
@@ -197,7 +197,10 @@ export class TypeScriptPlugin {
 
       fs.copySync(
         path.resolve('node_modules'),
-        path.resolve(path.join(BUILD_FOLDER, 'node_modules'))
+        path.resolve(path.join(BUILD_FOLDER, 'node_modules')),
+        {
+          dereference: true,
```

Comment:
❤️ Btw, I think LOC 177 needs to have this change too (at least, in our case we needed to patch both places)

Reply:
Can confirm, had to update both places.